frontend/public/components/conditions.tsx

```diff
@@ -20,7 +20,7 @@ export const Conditions: React.SFC<ConditionsProps> = ({ conditions }) => {
         <CamelCaseWrap value={condition.reason} />
       </div>
       {/* remove initial newline which appears in route messages */}
-      <div className="hidden-xs col-sm-5 col-md-4 co-break-word co-pre-line">
+      <div className="hidden-xs col-sm-5 col-md-4 co-break-word co-pre-line co-line-clamp">
```

Comment:
We probably need to show more than 3 lines, which is the default for `co-line-clamp`. These messages tend to be important when they're long. I'd say 5 minimum, but 10 might be better. We're just trying to avoid the case where it totally blows up the layout as in the Bugzilla.

https://github.com/spadgett/console/blob/3692933fb86c9acc75387e27fb78ae48d5e45437/frontend/public/style/mixin/_line-clamp.scss#L1-L6

You'll need a new CSS class that calls the mixin

```
.co-conditions__message {
  @include co-line-clamp(10);
}
```

Reply:
All set!